controllers/release/adapter.go

```diff
@@ -561,33 +657,82 @@ func (a *adapter) validateAuthor() *controller.ValidationResult {
 
 // validateProcessingResources will ensure that all the resources needed to process the Release exist.
 func (a *adapter) validateProcessingResources() *controller.ValidationResult {
-	resources, err := a.loader.GetProcessingResources(a.ctx, a.client, a.release)
+	releasePlan, err := a.loader.GetReleasePlan(a.ctx, a.client, a.release)
 	if err != nil {
-		if resources == nil || resources.ReleasePlanAdmission == nil || errors.IsNotFound(err) {
+		if errors.IsNotFound(err) {
 			a.release.MarkValidationFailed(err.Error())
 			return &controller.ValidationResult{Valid: false}
 		}
-
 		return &controller.ValidationResult{Err: err}
 	}
+
+	if releasePlan.Spec.Target != "" {
```

Comment:
This relates to the other comment about if we use target == nil or pipeline in RP == nil to determine if it is a tenant pipeline setup. Whichever you decide to go with, just make sure this line syncs up please

Reply:
changed to use  the pipeline one.